crates/matrix-sdk-crypto/src/olm/account.rs

```diff
@@ -1305,12 +1305,22 @@ impl Account {
                 // we might try to create the same session again.
                 // TODO: separate the session cache from the storage so we only add
                 // it to the cache but don't store it.
-                store
-                    .save_changes(Changes {
-                        sessions: vec![result.session.clone()],
-                        ..Default::default()
-                    })
-                    .await?;
+                let mut changes =
+                    Changes { sessions: vec![result.session.clone()], ..Default::default() };
```

Comment:
A bit whitespace wouldn't hurt here.
```suggestion
                    Changes { sessions: vec![result.session.clone()], ..Default::default() };

```

Reply:
I'm not seeing any difference between your suggestion and what's there.